.github/workflows/check.yml

```diff
@@ -0,0 +1,20 @@
+name: check
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install prereq
+      run: |
+        brew install docker tfenv tflint
+        tfenv install
```

Comment:
I think this is failing because tfenv is looking for a `.terraform_version` file

Reply:
added